PhotoMap/Features/MapPhoto/ViewModels/MapPhotoViewModelType.swift

```diff
@@ -19,6 +19,10 @@ protocol MapPhotoViewModelInput {
 protocol MapPhotoViewModelOutput {
     var isHiddenCategoryPicker: Bool { get }
     var categoryPublisher: Category? { get }
+    var photo: Photo { get }
+    var doneButtonTitle: String? { get }
+    var cancelButtonTitle: String? { get }
+    var closeCategoryPickerViewButtonTitle: String? { get }
```

Comment:
Don't forget to include tests for new outputs

Reply:
add tests 6eb9f57